history/cultures/annihilan.txt

```diff
@@ -15,6 +15,10 @@
 	discover_innovation = innovation_plenary_assemblies 
 	discover_innovation = innovation_crop_rotation 
 	#
+	discover_innovation = innovation_taming_the_skies
```

Comment:
I think Annihilans are pretty big to be considered ``massive_soldiers``. They should not be considered as giant also because they all have roughly the same size.